File: C++/1582.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int a,b,c;
    while(cin >> a >> b >> c){
        vector<int> l = {a,b,c};
        sort(l.begin(), l.end());
        
        if ((pow(l[2], 2) == pow(l[1], 2) + pow(l[0], 2)) && gcd(l[0], gcd(l[1], l[2])) == 1)
            cout << "tripla pitagorica primitiva\n";
        else if ((pow(l[2], 2) == pow(l[1], 2) + pow(l[0], 2)))
            cout << "tripla pitagorica\n";
        else
            cout << "tripla\n";
    }
}

File: Python/1582.py
from math import gcd


while True:
    try:
        l = list(map(int, input().split()))
        l.sort(reverse=True)
        
        if (l[0]**2 == l[1]**2 + l[2]**2) and gcd(l[0], gcd(l[1], l[2])) == 1:
            print('tripla pitagorica primitiva')
        elif (l[0]**2 == l[1]**2 + l[2]**2):
            print('tripla pitagorica')
        else:
            print('tripla')
    except EOFError:
        break